demos/manim/README.md

```diff
@@ -0,0 +1,44 @@
+# Manim Proof of Concept 
+
+Here you can create a docker container to aide the development of Manim animations. 
+
+## Building the Docker Container 
+
+To build the container run:
+
+```bash
+docker build . -t manim-poc
+```
+
+## Building Specific Animations 
+
+It will be easiest to specify which animation you want to run by first accessing the command line of the container and then running a manim function to specify animations.
+
+```bash
+docker run -it --rm -v $PWD:/code -w /code manim-1 /bin/bash
+```
+Then:
+```bash
+manim -pql manimTest.py Example_2
```

Comment:
Minor: Just to keep it aligned with the sample code, we can use `manim_examples.py` here.